src/main/java/school/lemon/changerequest/java/generics/pair/Pair.java

```diff
@@ -0,0 +1,38 @@
+package school.lemon.changerequest.java.generics.pair;
+
+/**
+ * Created by Yaroslav Pavlinskiy on 22.12.2016.
+ */
+public class Pair <K,V> {
+    protected V value;
+    protected K key;
+
+    public Pair(K key,V value) {
+        this.value = value;
+        this.key = key;
+    }
+
+    public V getValue() {
+        return value;
+    }
+
+    public void setValue(V value) {
+        this.value = value;
+    }
+
+    public K getKey() {
+        return key;
+    }
+
+    public void setKey(K key) {
+        this.key = key;
+    }
+
+    public boolean contains(K key)
```

Comment:
This method is quite weird, cause it not really check whether given pair `contains` smth. It just check whether some `key` is exactly same object as `key` from the pair. (also, you should not use `==`, but use `equals` instead).